src/translations/pages/assistant.ts

```diff
@@ -0,0 +1,39 @@
+import { translation as _ } from '@atb/translations/commons';
+
+export const Assistant = {
+  title: _('Planlegg reisen', 'Plan travel', 'Planlegg reisen'),
+  shortTitle: _('Reiseplanlegger', 'Assistant', 'Reiseplanlegger'),
```

Comment:
![Uploading 2023-10-24 at 11.21.56.png…]()

Reply:
I can't open the image 🤔

Reply:
![image](https://github.com/AtB-AS/planner-web/assets/14045515/abb3d581-533a-4c32-a4dc-50eaacb6c74f)